companion/src/firmwares/opentx/opentxinterface.cpp

```diff
@@ -30,8 +30,11 @@
 
 using namespace Board;
 
-#define OPENTX_FIRMWARE_DOWNLOADS        "https://downloads-22.open-tx.org/firmware"
-#define OPENTX_NIGHT_FIRMWARE_DOWNLOADS  "https://downloads-22.open-tx.org/nightlies/firmware"
+const char * const OPENTX_FIRMWARE_DOWNLOAD_URL[] = {
+  "https://downloads.open-tx.org/2.2/firmware",
```

Comment:
Could you change that to https://downloads.open-tx.org/2.2/release/firmware that would be cleaner

Reply:
I was too much conservative :)

Reply:
For SD may I do the same?